pipelines/internal/commands/run/run.go

```diff
@@ -350,9 +358,25 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		return nil, errors.New("no command or input file was specified")
 	}
 
-	zones, err := expandZones(project, listOf(*zones))
+	if *zones != "" && *regions != "" {
+		return nil, errors.New("both zones and regions are supplied")
+	}
+
+	var localeType LocaleType
+	var input *string
+	if *zones != "" {
+		localeType = Zone
+		input = zones
+	} else if *regions != "" {
+		localeType = Region
+		input = regions
+	} else {
+		return nil, errors.New("neither zones nor regions were supplied")
```

Comment:
As noted in the snippet above, I think we should maintain a default value rather than failing this case.

Reply:
Done, although defaulted zone instead - just to keep it the way it was. It doesn't really matter which one it should be, and from GVT side won't matter since caller will be forced to supply it anyways.